stable/cluster-autoscaler/templates/serviceaccount.yaml

```
@@ -2,6 +2,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
+  annotations: {{ toYaml .Values.rbac.serviceAccount.annotations | nindent 4 }}
```

Should this be the following instead? I'm never quite sure how whitespace gets handled.

```suggestion
  annotations:
    {{- toYaml .Values.rbac.serviceAccount.annotations | nindent 4 }}
```

---

We should also probably avoid setting `annotations: {}`